test/api/transport/transport.js

```diff
@@ -161,75 +87,51 @@ describe('RPC', function () {
 
 				it('should reject requests with incorrect authKey', function (done) {
 					validAcceptRequest.authKey = 'incorrect authKey';
-					clientSocket.wampSend('acceptPeer', validAcceptRequest)
-						.then(function (err, res) {
+					clientSocket.wampSend('updatePeer', validAcceptRequest)
+						.then(function () {
 							done('should not be here');
 						})
 						.catch(function (err) {
 							node.expect(err).to.equal('Unable to access internal function - Incorrect authKey');
 							done();
 						});
 				});
-			});
-		});
-
-		describe('removePeer', function () {
-			var validPeer;
-			var validRemoveRequest;
-
-			beforeEach(function () {
-				validRemoveRequest = {
-					authKey: 'authentication key',
-					peer: randomPeer
-				};
-				validPeer = _.clone(randomPeer);
-			});
-
-			describe('schema', function () {
-
-				it('should reject empty request', function (done) {
-					clientSocket.wampSend('removePeer', undefined)
-						.then(function (err, res) {
-							done('should not be here');
-						})
-						.catch(function (err) {
-							node.expect(err).to.equal('Expected type object but found type undefined');
-							done();
-						});
-				});
 
-				it('should reject requests without peer field defined', function (done) {
-					delete validRemoveRequest.peer;
-					clientSocket.wampSend('removePeer', validRemoveRequest)
-						.then(function (err, res) {
+				it('should reject requests without updateType', function (done) {
+					delete validAcceptRequest.updateType;
+					clientSocket.wampSend('updatePeer', validAcceptRequest)
+						.then(function () {
 							done('should not be here');
 						})
 						.catch(function (err) {
-							node.expect(err).to.equal('Missing required property: peer');
+							node.expect(err).to.equal('Missing required property: updateType');
 							done();
 						});
 				});
 
-				it('should reject requests without authKey field defined', function (done) {
-					delete validRemoveRequest.authKey;
-					clientSocket.wampSend('removePeer', validRemoveRequest)
-						.then(function (err, res) {
-							done('should not be here');
-						})
-						.catch(function (err) {
-							node.expect(err).to.equal('Missing required property: authKey');
-							done();
-						});
+				it('should reject requests when updateType is not a number', function (done) {
+					var nonNumbers = [{}, [], 'A', '1', NaN, true];
```

Comment:
- With number, do you mean integer or float?
- I would add `null, undefined, Infinity, false, empty string, float and negatives` if they are not supported. Here I have a great set
https://github.com/LiskHQ/lisk/blob/678-functional_tests_rewrite/test/functional/http/post/shared.js